src/driver/expo/ExpoQueryRunner.ts

```diff
@@ -251,6 +251,7 @@ export class ExpoQueryRunner extends AbstractSqliteQueryRunner {
                     fail(err)
                 },
                 () => {
+                    this.transactionDepth -= 1
```

Comment:
not sure how correct is current transactions implementation in the expo. `query` executes every query in transaction, and it's not clear what exactly `startTransaction` / `rollbackTransaction` methods call do in this method, also it's confusing why `commitTransaction` is missing in there.